src/lib/Preview.js

```diff
@@ -905,6 +905,9 @@ class Preview extends EventEmitter {
             case 'notificationhide':
                 this.ui.hideNotification();
                 break;
+            case 'previewnextfile':
```

Comment:
I'm thinking this should be `mediaend` related and not `previewnextfile` because in the future we may not want to preview the next file if it is not a media file.

Reply:
Generally, the message you broadcast should be for what happened, not what you want to happen, and then the subscriber then determines what it wants to do.

I'm thinking something like... `mediaendautoplay`? `mediaend` to me could be fired as an event that has nothing to do with autoplay, so appending autoplay makes it a bit more specific